datamap/mv/core/src/main/scala/org/apache/carbondata/mv/rewrite/DefaultMatchMaker.scala

```diff
@@ -121,8 +121,10 @@ object SelectSelectNoChildDelta extends DefaultMatchPattern with PredicateHelper
     } else if (subsumee.asInstanceOf[Select].outputList.contains(exprE)) {
       exprE match {
         case a@Alias(_, _) =>
-          exprListR.exists(a1 => a1.isInstanceOf[Alias] &&
-                                 a1.asInstanceOf[Alias].child.semanticEquals(a.child)) ||
+          exprListR
+            .exists(a1 => a1.isInstanceOf[Alias] &&
+                          a.name.equalsIgnoreCase(a1.asInstanceOf[Alias].name) &&
```

Comment:
Please add comment in what case this check is needed

Reply:
when semantic returns true then why we still need to check equalsignorecase

Reply:
ya, this was some  intermediate developement code, issed to remove, i ll update